test/pageobjects/CameraPermissions.js

```diff
@@ -1,16 +1,14 @@
-import Base from './BasePage.js'
-import {locale, verifyElementCopy} from '../utils/mochaw'
+import {verifyElementCopy} from '../utils/mochaw'
+import Screen from './Screen.js'
```

Comment:
Why renaming it to `Screen`? It doesn't give me the idea of something shared between multiple components.

Reply:
`Screen` doesn't represent a particular screen but something higher-level that other test classes can inherit from.